src/tests.py

```diff
@@ -250,6 +250,14 @@ class CircuitBreakerConfigurationTestCase(object):
     Tests for the CircuitBreaker class.
     """
 
+    def test_default_state(self):
+        """CircuitBreaker: it should get initial state from state_storage.
+        """
+        for state in (STATE_OPEN, STATE_CLOSED, STATE_HALF_OPEN):
+            storage = CircuitMemoryStorage(state)
+            breaker = CircuitBreaker(state_storage=storage)
+            self.assertEqual(breaker._state.name, state)
```

Comment:
would it make sense to just use `breaker.state` here? or is this intentional to minimize the code under test?

Reply:
Yeah, I think your suggestion would simplify the test a bit and stick to using public methods.